modules/statistics/php/stats_behavioural.class.inc

```diff
@@ -63,18 +63,23 @@ class Stats_Behavioural extends \NDB_Form
     {
         parent::setup();
 
-        $DB =& \Database::singleton();
+        $DB   =& \Database::singleton();
+        $user = \User::singleton();
```

Comment:
```suggestion
        $user = \NDB_Factory::singleton()->user();
```

Similar to above comment - not blocking but this would be an ideal way to use this object.